### PR TITLE
Add README to trace and metric exporter

### LINKED ISSUE #21: Add Readme.md for trace exporter.
Readme should contain 
- steps to install
- Usage Example,
- Link to full Example.

### LINKED ISSUE #22: Add Readme.md for monitoring exporter
Readme should contain.
 - steps to install
- Usage Example,
- Link to full Example.

### PR DESCRIPTION
**Pull request by ymotongpoo**
- Fixes #21 
- Fixes #22 
- Change all go.mod to use Go 1.14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
 
-Provides OpenTelemetry Exporters for Google Cloud Operations. 
+This repository contains the source code of 2 packages of OpenTelemetry exporters to [Google Cloud Trace](https://cloud.google.com/trace) and [Google Cloud Monitoring](https://cloud.google.com/monitoring).
 
-## Installation
+## OpenTelemetry Google Cloud Trace Exporter
 
+OpenTelemetry Google Cloud Trace Exporter allow the user to send collected traces and spans to Google Cloud.
+
+[Google Cloud Trace](https://cloud.google.com/trace) is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problemts in microservice architecture as well as monolithic architecture. It manages both the collection and lookup of gathered trace data.
```
Comment (james-bebbington) [inline README.md:9]:
```suggestion
[Google Cloud Trace](https://cloud.google.com/trace) is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problems in microservice & monolithic architectures. It manages both the collection and lookup of gathered trace data.
```

Comment (james-bebbington) [reply README.md:9]: Probably should replace this paragraph with a link to the trace/README.md file, as is done for metrics below? (currently this paragraph is replicated in both files)

File: exporter/metric/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Usage
+
+Add `github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric` to the import list and set up `go.mod` file accordingly.
+
+```go
+import mexpoter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
+```
+
+Once you import the metric exporter package, then register the exporter to the application, and start sending metrics. If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instruction in [Authentication](#Authentication).
```
Comment (james-bebbington) [inline exporter/metric/README.md:22]:
```suggestion
After you import the metric exporter package, then register the exporter to the application, and start sending metrics. If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in [Authentication](#Authentication).
```
(similar for traces README)

File: README.md (hunk 1 above)
Comment (ymotongpoo) [reply README.md:9]: I agree. I'll move the 3rd paragraph to Trace Exporter README

**Comment by ymotongpoo** [comment]
@james-bebbington Thank yo for reviewing. I changed re-organized the root README and fixed wordings in trace and metric exporter READMEs.

**Pull request merged**